content/en/docs/components/training/mxnet.md

```diff
@@ -101,18 +101,18 @@ should be created for each replica.
 
 ## Creating a TVM tuning job (AutoTVM)
 
-[TVM](https://docs.tvm.ai/tutorials/) is a end to end deep learning compiler stack, you can easily run AutoTVM with mxnet-operator.
+[TVM](https://docs.tvm.ai/tutorials/) is a end to end deep learning compiler stack, you can easily run AutoTVM with MXJob.
 You can create a auto tuning job by define a type of MXTune job and then creating it with
 
 ```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/tf-operator/master/examples/mxnet/tune/mx_job_tune_gpu_v1.yaml
+kubectl create -f https://raw.githubusercontent.com/kubeflow/training-operator/master/examples/mxnet/tune/mx_job_tune_gpu_v1.yaml
 ```
 
 Before you use the auto-tuning example, there is some preparatory work need to be finished in advance.
 To let TVM tune your network, you should create a docker image which has TVM module.
 Then, you need a auto-tuning script to specify which network will be tuned and set the auto-tuning parameters.
 For more details, please see [tutorials](https://docs.tvm.ai/tutorials/autotvm/tune_relay_mobile_gpu.html#sphx-glr-tutorials-autotvm-tune-relay-mobile-gpu-py).
-Finally, you need a startup script to start the auto-tuning program. In fact, mxnet-operator will set all the parameters as environment variables and the startup script need to reed these variable and then transmit them to auto-tuning script.
+Finally, you need a startup script to start the auto-tuning program. In fact, MXJob will set all the parameters as environment variables and the startup script need to reed these variable and then transmit them to auto-tuning script.
```

Comment:
Fixes in bold below.
Finally, you need a startup script to start the auto-tuning program. In fact, MXJob will set all the parameters as environment variables and the startup script **needs** to **read** these variable and then transmit them to **the** auto-tuning script.